src/consts/models.tsx

```diff
@@ -191,6 +212,18 @@ export interface Wallet {
   getScriptHashes: () => string[];
   getAddressForTransaction: () => string;
   password?: string;
+  pubKeys?: Buffer[];
+  getDerivationPath: () => string;
+}
+
+export interface WalletPayload {
+  name: string;
+  gap_limit: number;
+  derivation_path?: {};
```

Comment:
![Zrzut ekranu 2021-01-13 o 09 13 52](https://user-images.githubusercontent.com/67273035/104425345-dbfe8f00-5580-11eb-8bce-18b3c0ae44db.png)

Reply:
yes, the same warning pop on 'Function' type.